Error while compiling Godot [Windows x86_32]

### Tested versions

Reproducable after PR #91201 

### System information

Win 11 64bit - v4.4.custom_build

### Issue description

SCons terminates building Godot for Windows **on x86_32 archs only** with the following error:

`platform\windows\os_windows.cpp(990): error C2664: 'BOOL InitializeProcThreadAttributeList(LPPROC_THREAD_ATTRIBUTE_LIST,DWORD,DWORD,PSIZE_T)' : argument 4 cannot be converted from value  'size_t *' to value 'PSIZE_T' `

Error seems to be caused from file `os_windows.cpp` after #91201 has been merged. x86_64 archs seems to compile without any error

### Steps to reproduce

Compile Godot Windows x86_32 using the master branch

SCons will terminate the building with the mentioned error

### Minimal reproduction project (MRP)

N/A

--- a/platform/windows/os_windows.cpp
+++ b/platform/windows/os_windows.cpp
@@ -986,7 +986,7 @@ Dictionary OS_Windows::execute_with_pipe(const String &p_path, const List<String
 	pi.si.StartupInfo.hStdOutput = pipe_out[1];
 	pi.si.StartupInfo.hStdError = pipe_err[1];
 
-	size_t attr_list_size = 0;
+	SIZE_T attr_list_size = 0;
 	InitializeProcThreadAttributeList(nullptr, 1, 0, &attr_list_size);
 	pi.si.lpAttributeList = (LPPROC_THREAD_ATTRIBUTE_LIST)alloca(attr_list_size);
 	if (!InitializeProcThreadAttributeList(pi.si.lpAttributeList, 1, 0, &attr_list_size)) {
@@ -1082,7 +1082,7 @@ Error OS_Windows::execute(const String &p_path, const List<String> &p_arguments,
 			pi.si.StartupInfo.hStdError = pipe[1];
 		}
 
-		size_t attr_list_size = 0;
+		SIZE_T attr_list_size = 0;
 		InitializeProcThreadAttributeList(nullptr, 1, 0, &attr_list_size);
 		pi.si.lpAttributeList = (LPPROC_THREAD_ATTRIBUTE_LIST)alloca(attr_list_size);
 		if (!InitializeProcThreadAttributeList(pi.si.lpAttributeList, 1, 0, &attr_list_size)) {

